src/Features/Core/Portable/Microsoft.CodeAnalysis.Features.csproj

```diff
@@ -99,6 +99,7 @@
     <RestrictedInternalsVisibleTo Include="Microsoft.VisualStudio.IntelliCode.CSharp" Partner="Pythia" Key="$(IntelliCodeKey)" />
     <RestrictedInternalsVisibleTo Include="Microsoft.VisualStudio.IntelliCode.CSharp.Extraction" Partner="Pythia" Key="$(IntelliCodeKey)" />
     <InternalsVisibleTo Include="IdeCoreBenchmarks" />
+    <InternalsVisibleTo Include="dotnet-watch" Key="$(AspNetCoreKey)" />
```

Comment:
❗ new external IVTs are not allowed. A new external access assembly needs to be created in a folder [here](https://github.com/dotnet/roslyn/tree/main/src/Tools/ExternalAccess) (presumably a **DotnetWatch** folder), and IVTs can be provided from that assembly to dotnet-watch.